pkg/private/bundle/bundle.go

```diff
@@ -497,9 +484,11 @@ func Podspec(task Task, ref name.Reference, arch, mFamily, sa, ns string, gvisor
 		},
 	}
 
-	if gvisor {
-		c := "gvisor"
-		pod.Spec.RuntimeClassName = &c
+	for k, v := range anns {
+		if _, ok := pod.Annotations[k]; ok {
+			return nil, fmt.Errorf("annotation %q already set", k)
+		}
+		pod.Annotations["melange.chainguard.dev/"+k] = v
```

Comment:
Above checks `k` but this sets `"melange.chainguard.dev/"+k` is that intentional?

Reply:
Not intentional, fixed.